Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -77,28 +77,28 @@ public LayerMask[] MagneticSurfaces
 
         [SerializeField]
         [Tooltip("Max distance for raycast to check for surfaces")]
-        private float maxDistance = 3.0f;
+        private float maxRaycastDistance = 50.0f;
```

Comment:
>50.0 [](start = 43, length = 4)

I think 50 is further than the default raycast distance of 10 for things like the hand ray right?

Reply:
@wiwei  Yes there is actually a related issue #5269 

Although I don't understand why 10m? That's not very useful for VR and you could have large items in a scene

It seems like something people will keep running into this and it's not obvious how to debug. The SurfaceMagnetism in the example scene I think was actually broken because of this property being set to 3m.

Think of it this way:

If I try to use the component out-of-box and raycast to a wall 20m away (AR or VR or hands or head or other raycast type), it doesn't work. It's not clear where to debug and why it doesn't work

If I can use my hands to raycast a small item to a wall 50m away and it's not very accurate or useful for my scenario, it's more obvious that I just need to limit my raycast to certain layermask or to limit distance